Mathlib/RingTheory/Localization/Integral.lean

```diff
@@ -466,3 +466,35 @@ lemma isAlgebraic_of_isFractionRing {R S} (K L) [CommRing R] [CommRing S] [Field
     apply IsIntegral.tower_top (R := R)
     apply IsIntegral.map (IsScalarTower.toAlgHom R S L)
     exact Algebra.IsIntegral.isIntegral (s : S)
+
+namespace Polynomial
+
+lemma isLocalization {R} [CommRing R] (S : Submonoid R) (A) [CommRing A] [Algebra R A]
+    [IsLocalization S A] :
+    letI := (mapRingHom (algebraMap R A)).toAlgebra; IsLocalization (S.map C) A[X] := by
+  letI := (mapRingHom (algebraMap R A)).toAlgebra
+  constructor
+  · rintro ⟨_, r, hr, rfl⟩
+    simpa [RingHom.algebraMap_toAlgebra] using (IsLocalization.map_units A ⟨r, hr⟩).map C
+  · intro z
+    obtain ⟨b, hb⟩ := IsLocalization.integerNormalization_spec S z
+    refine ⟨⟨IsLocalization.integerNormalization S z, _, b, b.2, rfl⟩, ?_⟩
+    ext i
+    simp only [RingHom.algebraMap_toAlgebra, coe_mapRingHom, map_C, coeff_mul_C, coeff_map, hb,
+      mul_comm, Algebra.smul_def]
+  · intros x y e
```

Comment:
```suggestion
  · intro x y e
```

Reply:
Isn't this precisely the use case of `ìntros`? I don't really see the point of changing here

Reply:
At this point, I would rather wait for the `intro ..` syntax to replace `intros` rather than thinking about the color of the bikeshed myself